src/Compilers/CSharp/Portable/Syntax/CompilationUnitSyntax.cs

```diff
@@ -19,16 +20,22 @@ public IList<ReferenceDirectiveTriviaSyntax> GetReferenceDirectives()
 
         internal IList<ReferenceDirectiveTriviaSyntax> GetReferenceDirectives(Func<ReferenceDirectiveTriviaSyntax, bool>? filter)
```

Comment:
Very unfortunate that `GetReferenceDirectives` is a public API.

That being said, another thing we should do is change what we're using `GetReferenceDirectives()` for. The only two calls to the public `GetReferenceDirectives()` API above are checking `GetReferenceDirectives().Count > 0`. We should be able to change those to avoid needing to construct this list entirely.

Reply:
agreed.  i'll see about making that change as well.

Reply:
done.